.semaphore/semaphore.yml

```diff
@@ -42,31 +41,32 @@ blocks:
       agent:
         machine:
           type: a1-standard-4
-          os_image: macos-mojave-xcode11
+          os_image: macos-xcode11
       secrets:
         - name: macos-ci-pubkey
-        - name: check-deps-script
         - name: ignore-ci-homebrew
       prologue:
         commands:
           - echo Trying to restore cache...
           - ssh-keyscan -p 29920 66.185.20.68 >> ~/.ssh/known_hosts
           - chmod 400 ~/.ssh/semaphore_cache_key
-          - cache restore new-macports-prefix
-          - sudo chmod +x /Users/semaphore/check-install-macports.sh
-          - export PATH="/Users/semaphore/local/bin:/Users/semaphore/local/sbin:$PATH"
-          - mkdir -p ./local
+          - export PATH=/opt/local/bin:/opt/local/sbin:$PATH
+          - export MANPATH=/opt/local/share/man:$MANPATH
+          - export DISPLAY=:0.0
+          - cache restore macports-10_15
+          - mkdir -p /Users/semaphore/local
+          - mkdir -p /Users/semaphore/Applications
+          - sudo ln -sf /Users/semaphore/local/Applications /Users/semaphore/Applications/MacPorts
           - sudo ln -sf /Users/semaphore/local/ /opt/local
-          - test -e /Users/semaphore/local/.deps_installed || /Users/semaphore/check-install-macports.sh | cat 2>&1
+          - test -e /Users/semaphore/local/bin/port || ( curl -LJ https://www.dropbox.com/s/4c5oav6za5l4gvv/macports-10_15-install.tar.gz?dl=1 | tar x ) 
```

Comment:
why is macport on a dropbox instead of some "official" repository?

Reply:
It's my Dropbox hahaha, it's because it's the already built ports; to save time essentially.

Reply:
Hmm maybe create a repo + release for this containing that tarball? This way it's listed on github, saving you space on dropbox and for everyone to see?